sdk/src/types/block/output/output_id_proof.rs

```diff
@@ -194,7 +194,7 @@ impl Packable for LeafHash {
     }
 }
 
-/// Value Hash contains the hash of the value for which the proof is being computed.
+/// Contains the hash of the value for which the proof is being computed.
```

Comment:
You have added `OutputCommitmentProof` in the 2 other comments, do you want to add here for consistency? Resolve if not.

Reply:
Done